spec/components/schemas/2fa/configuration/requests/website.ts

```diff
@@ -0,0 +1,37 @@
+import { SchemaObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../../../../utils/ref';
+
+const website: SchemaObject = {
+  title: 'Website domains',
+  description: 'Needded information to use this API two factor authentication service using a website or a mobile app',
```

Comment:
Acho que ficaria melhor:
`Information needed to use this two factor authentication API service on a website or on a mobile app.`